code-challenges.test.js

```diff
@@ -1,59 +1,223 @@
+// //       expect(object(num, expectedOutput)).toEqual(expectedOutput)
+// //     })
+// //   })
+
+
+// // //     // function isDivisibleBy3(num){ if ((num % 3) == 0){ return true; } else { return false; } };
+
+// // // // When working on this all my work came back undefined, which is typically what we want but when doing my function is not working and that object wasnt defined . I have to remember to read the question for Context. I went through the syllabus not thinking when you meant object you meant object. 
+// // // I find myself thinking really hard, into the jest doing its own math, when we give it what to say and think. My mind is overflowing with console.log. 
+
+// // _________________________________________________________________________________________________________________________________________________________
+
+
+// // // --------------------2) 
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // // // // // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // // // // // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+
+// // Create a function that takes in an array of words and returns an array with all the words capitalized.
+
+
+// // // a) Create a test with expect statements for each of the variables provided.
+
+describe("randomNouns", () => {
+    it("an array with all the words capitalized", () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+      expect(randomNouns(randomNouns1)).toUpperCase("STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW")
+      expect(randomNouns(randomNouns2)).toUpperCase("TEMPERATURE", "DATABASE", "CHOPSTICKS", "MANGO")
+    })
+  })
+
+//   const randomNouns = array.map ((array) => {
+//     return array(array.toUpperCase())
+// })
+
+const randomNouns = (array) => {
+    array.map((array) => {
+        return array(array())
+   })
+  }
+
+  
+
+//   //  output: 
+// //   ReferenceError: randomNouns is not defined
+
+// // Pseudo Code:
+// // create a function that takes in an array of words and returns array with all the words capitalized
+// // // declare a function that takes in an array
+// // const randomNouns = (array) => {
+// //   // return the array output from the map action
+// //   return array.map((array) => {
+// //     // with each value access the first index and capitalize the letter then  the remainder of the word
+// //     return array.toUpperCase()
+// //   })
+// // }
```

Comment:
This is getting super close.  The last step you would need is to break up the first letter in the string to capitalize it, then join it back to the rest of the word.